content/en/docs/contributing/pr-checks.md

```diff
@@ -59,51 +57,51 @@ If any issues are found, annotations are added to your files in the
 alternative, you can run `npm run check:text -- --fix` locally to fix most
 issues. Run `npm run check:text` again and manually fix the remaining issues.
 
-### MARKDOWN linter
+### `MARKDOWN linter` {.notranslate lang=en}
 
 This check verifies that
 [standards and consistency for Markdown files are enforced](../style-guide/#markdown-standards).
 
 If any issues are found, run `npm run:format` to fix most issues. For more
```

Comment:
This is the other one I was questioning....

Reply:
Good catch! Fixed.